icons/ktor/ktor-plain-wordmark.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg">
+  <path d="m22.855 43.2-7.238 7.241-6.472 6.473 13.21 13.215V56.414H36.07L22.855 43.2zM59.43 54.647v18.286h4.023v-5.59l2.117-2.192 5.668 7.782h4.832l-7.785-10.528 7.473-7.758H70.82l-7.367 7.997v-7.997H59.43zm18.13.625v3.579h-1.67v3.394h1.672v6.637c0 3.238 1.644 4.207 4.074 4.207a5.859 5.859 0 0 0 3.136-.813v-3.183a4.04 4.04 0 0 1-1.96.496c-.887 0-1.28-.445-1.28-1.36v-5.984h3.291v-3.394h-3.29v-3.579H77.56zm-54.205 2.141V70.13H36.07V57.414H23.355zm13.715 0V71.13H23.355l6.332 6.326 6.883 6.887 13.715-13.715L37.07 57.414zm56.684 1.098c-4.363 0-7.602 3.265-7.602 7.289v.05c0 4.025 3.215 7.239 7.551 7.239 4.363 0 7.602-3.265 7.602-7.29v-.05c0-4.023-3.215-7.238-7.551-7.238zm17.68.156c-2.084.057-3.277 1.273-4.036 3.086v-2.82h-3.972v14h3.972v-5.172c0-3.344 1.618-4.938 4.258-4.938h.207v-4.152a5.927 5.927 0 0 0-.43-.004zm-17.73 3.266c2.194 0 3.683 1.804 3.683 3.867v.05c0 2.067-1.36 3.817-3.633 3.817-2.192 0-3.684-1.805-3.684-3.867v-.051c0-2.066 1.36-3.816 3.633-3.816z"/>
```

Comment:
Looks like you forgot to change the fill color here :)

Reply:
Yep, related to #1291